tripy/tests/flat_ir/ops/test_gather.py

```diff
@@ -16,28 +16,41 @@
 #
 
 import tripy as tp
-from tripy import int32
 
 from tripy.flat_ir.ops import DynamicGatherOp
 from tripy.frontend.trace import Trace
-
+import pytest
 
 class TestGatherOp:
-    def test_gather_str(self):
-        data = tp.Tensor([3.0, 4.0], name="data")
-        index = tp.Tensor([0], dtype=int32, name="indices")
-        out = tp.gather(data, 0, index)
+    @pytest.mark.parametrize("axis", [0,1,2])
+    def test_gather_str_dim0(self, axis):
```

Comment:
What does the suffix `dim0` signify?

Reply:
Was planning on doing separate test cases for each dimension tested. I will remove it.